src/components/Common/Typography/Link.tsx

```diff
@@ -14,15 +18,19 @@ function isLocalLink(href?: string | null) {
   )
 }
 
+const TARGET_BLANK = '_blank'
+
 export default function Link({ target, rel, href, onClick, className, ...props }: Props) {
   const isLocal = isLocalLink(href)
-  const linkTarget = !target && !isLocal ? '_blank' : target || undefined
+  const linkTarget = !target && !isLocal ? TARGET_BLANK : target || undefined
```

Comment:
This can also be written like
`const linkTarget = !isLocal ? target || TARGET_BLANK : undefined;`